routes/index.js

```diff
@@ -8,14 +13,52 @@ router.get('/', function(req, res) {
     });
 });
 router.post('/pushData', function(req, res, next) {
+    console.log("starting");
     var db = req.db;
     var resultsHolder = db.get('formResults');
-    resultsHolder.insert(req.body, function(err, result) {
-        if (err)
-            res.send('Error');
-        else
-            res.send('Success');
-    });
+    if(currentEmail == ''){
```

Comment:
You might consider using `if (!currentEmail)` for conciseness. The `!` coerces the following value into a boolean (true/false) and negates it. An empty string evaluates to false, but a string with content will be true.